projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.html

```diff
@@ -0,0 +1,25 @@
+<div
+  (click)="closeOffCanvas()"
+  [@fade]="opened ? 'in' : 'out'"
+  class="go-off-canvas-backdrop"
+  tabindex="-1"
+>
```

Comment:
I haven't decided yet. We haven't talked much about how we write HTML. This style of attribute wrapping is the default in prettier. It's hard for me to get used to honestly, but it does seem nice as the number of attributes we have to include grows. I like that it reads straight down regardless of how long the name of the component is.
 
My thought is:
```html
<!-- This feels a little harder to track because it zig zags -->
<some-component attribute-one="thing"
                attribute-two="thing">
</some-component>

<some-component-with-a-long-name attribute-one="thing"
                                 attribute-two="thing"
                                 attribute-three="thing"
                                 attribute-four="thing">
</<some-component-with-a-long-name>
```
```html
<!-- This feels a nicer to track because it is consistent -->
<some-component
  attribute-one="thing"
  attribute-two="thing"
>
</some-component>

<some-component-with-a-long-name
  attribute-one="thing"
  attribute-two="thing"
  attribute-three="thing"
  attribute-four="thing"
>
</<some-component-with-a-long-name>
```

Reply:
Hm.. this is actually not bad at all to read. We should talk about this as well in our next guild meeting 👍